### PR TITLE
Upgrade Pipelines Operator to 1.10

### PR DESCRIPTION
**Pull request by bnallapeta**
The PR brings in the below changes:
- Upgrades OpenShift Pipelines Operator to 1.10
- Removes Pipelines as Code files (independent installation) as PaC (version 0.17.x) comes with 1.10 operator
- Remove disabling PaC installation from TektonConfig
- Update Security tests to remove pipelines-as-code namespace as PaC components are now part of openshift-pipelines namespace.

### CODE REVIEW

File: operator/gitops/argocd/pipeline-service/kustomization.yaml [hunk 1]
```diff
@@ -4,7 +4,6 @@ kind: Kustomization
 
 resources:
   - openshift-pipelines
-  - pipelines-as-code
```
Comment (xinredhat) [inline operator/gitops/argocd/pipeline-service/kustomization.yaml:7]: @bnallapeta As we don't need to install Pac as a separate resource. Do we need to delete its related manifest [files](https://github.com/openshift-pipelines/pipeline-service/tree/main/operator/gitops/argocd/pipeline-service/pipelines-as-code)

Comment (bnallapeta) [reply operator/gitops/argocd/pipeline-service/kustomization.yaml:7]: yes, will do that later. That's why marked it as WIP. 
Wanted to get the tests triggered first.

Comment (bnallapeta) [reply operator/gitops/argocd/pipeline-service/kustomization.yaml:7]: Done.

File: operator/gitops/argocd/pipeline-service/kustomization.yaml [hunk 2]
```diff
@@ -4,7 +4,6 @@ kind: Kustomization
...
   - tekton-chains
```
Comment (Roming22) [inline operator/gitops/argocd/pipeline-service/kustomization.yaml:7]: I would expect chains to be installed by the operator as well since it's GA.

Comment (gabemontero) [reply operator/gitops/argocd/pipeline-service/kustomization.yaml:7]: +1
that was our understanding back during our last round of feature refinement, as installing it from the operator, aside from "reduced duplication", gets us "use RH based images" etc.

Comment (gabemontero) [reply operator/gitops/argocd/pipeline-service/kustomization.yaml:7]: in fact the test failure implies to me "let's get the chains installed via 1.10 to get versions correct:
```
- Public key: Error from server (BadRequest): error when creating "/source/operator/test/manifests/test/tekton-chains/public-key.yaml": admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "serviceAccountName"
command terminated with exit code 1
```

**Comment by Roming22** [comment]
Closing in favor of #612 and #613.

**Pull request closed without merging**